addons/test_website/models/model.py

```diff
@@ -35,6 +36,23 @@ def _search_get_detail(self, website, order, options):
             'icon': 'fa-check-square-o',
             'order': 'name asc, id desc',
         }
+        return result
+
+
+class TestSubModel(models.Model):
+    _name = 'test.submodel'
+    _description = 'Website Submodel Test'
+
+    name = fields.Char(required=True)
+    test_model_id = fields.Many2one('test.model', string="Parent model")
+    tag_id = fields.Many2one('test.tag', 'Tag')
```

Comment:
Isn't this how the framework / odoo will name it if omitted?
It will remove trailing `_id` or `_ids`, replace `_` by ` ` and uppercase it